src/constants.ts

```diff
@@ -71,6 +72,7 @@ export const PERSISTENCE_RESERVED_PROPERTIES = [
     ENABLED_FEATURE_FLAGS,
     USER_STATE,
     PERSISTENCE_EARLY_ACCESS_FEATURES,
+    PERSISTENCE_FEATURE_FLAG_DETAILS,
```

Comment:
If you're wondering why I am excluding this, if we don't, it would get rendered like so:

![Screenshot 2025-03-13 at 7 06 56 PM](https://github.com/user-attachments/assets/540699a7-7932-45a1-9fa0-2c69bae4d2ad)

Yikes! Useful info, but maybe too much and we'd need to figure out how to render it more nicely.

Reply:
yeah, super reasonable to not include all this info by default.  Would be awesome if we had a UI component to make this nice, but it's out of scope for this change IMO.